### PR TITLE
Background figures

### PR DESCRIPTION
**Pull request by daisieh**
I drew some figures that I thought might help add visuals to the git lessons.

### CODE REVIEW

**Comment by wking** [comment]
On Tue, Mar 17, 2015 at 01:33:40PM -0700, Daisie Huang wrote:
> I drew some figures that I thought might help add visuals to the git
> lessons.

The more pictures the better (in my opinion).

On the technical side, I'd rather avoid embedding raster content in
the SVGs, and it looks like you have embedded raster content in each
SVG except git_staging.svg.

For some Creative Commons Attribution-NonCommercial-ShareAlike 3.0
Unported examples of these sorts of graphics, see
https://github.com/progit/progit2, for example, [1].  SWC can't use
those directly, because we don't have the NonCommercial portion of the
license for our teaching material (and the PNGs are rasterized), but I
personally prefer these sorts of abstracted graphics to your more
three-dimensional git_staging.svg.  I don't have time to work up
alternatives though, so if my feedback doesn't sound convincing, or if
it does but you just don't have time to rework the graphics, I'm fine
with these being merged as they stand (but I'm not a git-novice
maintainer either ;).

[1]: https://github.com/progit/progit2/blob/master/book/02-git-basics/images/lifecycle.png

**Comment by daisieh** [comment]
Augh, the raster bits were left from Keynote putting them in the original graphics. They do nothing. I should be able to take them out.

**Comment by daisieh** [comment]
I don't know if we need any sort of extra attribution from me, but I can tell you that I drew these (very artistically advanced) graphics myself from scratch and don't much care about extra attribution. The git cartoon is something I put on figshare though, so I cited myself for that one.

**Comment by daisieh** [comment]
The cartoon is something that I thought might be helpful to leave up as a slide on the screen during exercises, so that people can have something concrete to look at as they work through the examples themselves.

**Comment by daisieh** [comment]
@wking, @iglpdc, any objections to these figures?

File: 01-setup.md [hunk 1]
```diff
@@ -18,6 +18,30 @@ version control is much better for this than this:
...
+
+![Multiple versions can be merged](fig/merge.svg)
+
+## How Git works
+
+A version control system is a tool that keeps track of these changes for us and helps us version and merge our files. A system like Git is designed to keep multiple sets of versions and changes in sync across different computers or servers. This is called a _distributed_ system.
+
+A [repository](reference.html#repository) is the set of files that we want to keep under version control.
+
+With Git, every user who wants to make changes to a repository has their own copy of the files in the repository, along with their own copy of the changes (the _commits_) that have been made to those files. Git keeps the commits in a secret directory along with the copies of the files.
```
Comment (iglpdc) [inline 01-setup.md]: Maybe "secret" to "hidden". It's the actual name and also demystifies what's happening.

Also you could link commits to their glossary entry in `reference.md`.

**Comment by iglpdc** [comment]
I like the figures. I think that you have an extra "line" in "play-changes" and "versions". The first version of the file has four lines, but it seems to me that it should have only the first three. 

Other than that and my comment on `01-setup.md`, +1 to merge.

**Comment by daisieh** [comment]
I think the idea was that it was a changed line, not an added line, but is that confusing?

**Comment by iglpdc** [comment]
Ok, no, I think it's fine: I wasn't sure either :-). I'm merging then.

**Comment by daisieh** [comment]
@iglpdc , are you merging this?

**Comment by iglpdc** [comment]
Sorry, @daisieh! Something really weird happened the other day. I clicked the green button to merge and  it got trapped in "Merging..." forever. I waited a couple of hours, tried to reload the page, etc,... and nothing worked. Then, the weekend came and I forgot :-) That's for reminding me and sorry for the wait.

**Comment by daisieh** [comment]
No problem! Thanks for the merge :)

**Pull request merged**